linkerd/app/inbound/src/lib.rs

```diff
@@ -202,14 +191,14 @@ impl Config {
             //     .cache(
             //         svc::layers().push_on_response(
             //             svc::layers()
-            //                 // If the service has been ready & unused for `cache_max_idle_age`,
-            //                 // fail it.
-            //                 .push_idle_timeout(cache_max_idle_age)
-            //                 // If the service has been unavailable for an extend time, eagerly
+            //                 // If the service has been unavailable for an extended time, eagerly
             //                 // fail requests.
             //                 .push_failfast(dispatch_timeout)
             //                 // Shares the service, ensuring discovery errors are propagated.
-            //                 .push_spawn_buffer(buffer_capacity)
+            //                 .push_spawn_buffer_with_idle_timeout(
+            //                     buffer_capacity,
+            //                     cache_max_idle_age,
+            //                 )
```

Comment:
Thanks for copying over the changes to commented out code as well, that will make my life much easier later :)